lighthouse-core/test/config/config-test.js

```diff
@@ -623,90 +683,171 @@ describe('Config', () => {
     });
   });
 
-  describe('generateNewFilteredConfig', () => {
+  describe('filterConfigIfNeeded', () => {
```

Comment:
the only changes to existing tests were
- to use `onlyCategories` (etc) instead of `Config.generateNewFilteredConfig` directly
- to use `requireAudits`/`requireGatherers` which now call the shorthand expansion methods instead of `expandAuditShorthandAndMergeOptions` directly
caused a little code churn to maintain the same test, but the intention was for test path/result to be exactly the same